.github/workflows/gh-workflow-approve.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Approve GitHub Workflows
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - synchronize
+
+jobs:
+  approve:
+    name: Approve ok-to-test
+    if: contains(github.event.pull_request.labels.*.name, 'ok-to-test')
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
```

Comment:
this makes me very nervous. github PR workflows have a history of privilege escalation issues

Reply:
OK, let me close it, 

The best approach is probably still to implement it on the prow side.